R/display.R

```diff
@@ -0,0 +1,71 @@
+
+
+
+
```

Comment:
Yer, my logic here was as we are using R6 on the backend it would be best to use it in all places where we deem it necessary to use an object for consistency. I would rather avoid a situation where we have half our stuff as S3 and the other half as R6.  Personally I also find R6 much easier to rationalise about and code (I guess cos its more familiar to OO in other languages)